tools/esp32/mk_qemu_img.sh

```diff
@@ -71,6 +72,8 @@ done
 # Make sure we have the required argument(s)
 
 if [ -z "${BOOTLOADER_IMG}" ] || [ -z "${PARTITION_IMG}" ] ; then
+  echo ""
+  echo "Failed to generate QEMU image!"
```

Comment:
Is it really necessary to output these lines?
I would let the "failure" only for the generation process itself. For the omission of parameters I think the USAGE output is already intuitive.

Reply:
I understand. Do you think that maybe adding ${SCRIPT_NAME} to the echo line below could help indicate that the issue is on the QEMU image generation?
`echo "${SCRIPT_NAME}: Missing bootloader and partition table binary images."`

Reply:
> echo "${SCRIPT_NAME}: Missing bootloader and partition table binary images."

Yeah, that works fine for me!